CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## [next]
 
 - Add `NumberBox` widget. ([#560](https://github.com/bdlukaa/fluent_ui/issues/560) [#771](https://github.com/bdlukaa/fluent_ui/pull/771))
+- Add `min`/`max` properties to the `NumberBox` widget. ([#789](https://github.com/bdlukaa/fluent_ui/pull/789))
+- Allow `NumberBox` Widget to accept int and double values and mathematical expressions. ([#789](https://github.com/bdlukaa/fluent_ui/pull/789))
```

Comment:
I am not sure we need to add these to the changelog since the version hasn't been published. Wdyt?

Reply:
Yep, I just added the reference to this PR in the Changelog